app.ts

```diff
@@ -0,0 +1,261 @@
+let headingColumns: any = document.querySelector("#column-headings-container"); // Headings
+let infoColumns: any = document.querySelector("#info-columns-container"); // Information
+let fromNumber: number = 0;
+let count: number = 0;
+let timeout: number = 0;
+
+window.onload = () => {
+  fromNumber = 0;
+};
+
+let debounce = (func: any, delay: number) => {
+  return function () {
+    clearTimeout(timeout);
+
+    timeout = setTimeout(() => {
+      func();
+    }, delay);
+  };
+};
+
+let createNavigation = () => {
+  let recordNav: any = document.querySelector("#record-navigation-container"); // Navigation area
+  recordNav.innerHTML = `
+  <div class="navigation-btns">
+    <button value="first" class="first-page-btn">First Page</button>  
+    <button value="previous" class="previous-records-btn">Previous</button>
+    <button value="next" class="next-records-btn">Next</button>
+    <button value="last" class="last-page-btn">Last Page</button>  
+    <button onclick="recordSelection()" id="confirmation-btn">Get Record</button>
+  </div>
+  <div class="current-page-container">
+    <p class=current-page></p>
+  </div>
+  `;
+};
+
+function recordSelection() {
+  let selectionArea: any = document.querySelector("#record-navigation-container");
+  let singleRecordSelection = `
+    <button id="return-btn">Return</button>
+      <div id="user-input-data">
+        <div class="navigation-input-area-id" id="id">
+          <label class="record-labels" for="record-id"
+            >Enter record ID :
+          </label>
+          <input
+            type="text"
+            min="0"
+            minlength="1"
+            maxlength="6"
+            name="record-id"
+            id="record-id"
+            class="navigation-input"
+            value="0"
+          />
+        </div>
+        <p class="amount-of-records"></p>
+      </div>
+    <button id="get-record-btn">See Record</button>
+    `;
+
+  selectionArea.innerHTML = "";
+  selectionArea.innerHTML = singleRecordSelection;
```

Comment:
Probably not necessary to clear it since you are anyway assigning it a new value that overwrites the previous one?